test/prism/ractor_test.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+return unless defined?(Ractor)
+
+require_relative "test_helper"
+
+return if Prism::TestCase.windows?
```

Comment:
I think this condition should be whether `fork` is implemented or not, instead of the platform name.

```ruby
return  unless defined?(fork)
```

Reply:
https://github.com/ruby/prism/pull/3505